terraform/modules/tf_alert_processor_iam/main.tf

```diff
@@ -0,0 +1,105 @@
+// Permissions specific to the alert processor: decrypting secrets, sending alerts to outputs
+
+locals {
+  firehose_arn_prefix = "arn:aws:firehose:${var.region}:${var.account_id}"
+  lambda_arn_prefix   = "arn:aws:lambda:${var.region}:${var.account_id}:function"
+}
+
+resource "aws_iam_role_policy" "base" {
+  name   = "DecryptSecretsAndUseDefaultOutputs"
```

Comment:
can we split `DecryptSecrets` and `DefaultOutputs` into two policies? I tend to follow the single responsibility model for policies in order to easily understand what an IAM role can do (via the UI especially)